src/sql/azdata.proposed.d.ts

```diff
@@ -1686,4 +1686,42 @@ declare module 'azdata' {
 		 */
 		objectType?: string;
 	}
+
+	export namespace window {
+		export interface Wizard {
+			/**
+			 * When true, the wizard will show a loading spinner at the bottom left of the footer.
+			 */
+			loading: boolean;
+
+			/**
+			 * This sets the alert text which gets announced when the loading spinner is shown.
+			 */
+			loadingText: string;
+
+			/**
+			 * The text to display while loading is set to false. Will also be announced through screen readers
+			 * once loading is completed.
+			 */
+			loadingCompleteText: string;
+		}
+
+		export interface Dialog {
+			/**
+			 * When true, the wizard will show a loading spinner at the bottom left of the footer.
+			 */
+			loading: boolean;
+
+			/**
+			 * This sets the alert text which gets announced when the loading spinner is shown.
+			 */
+			loadingText: string;
+
+			/**
+			 * The text to display while loading is set to false. Will also be announced through screen readers
+			 * once loading is completed.
+			 */
+			loadingCompleteText: string;
+		}
```

Comment:
I didn't notice this suggestion. I have made a loadingComponentBase interface because I thought it will be used by components which are not dialogs as well.

Reply:
Let me know if you want me to switch to this suggestion.

Reply:
What you have is fine, thanks!